addons/editor-theme3/theme3.js

```diff
@@ -615,6 +615,29 @@ export default async function ({ addon, console, msg }) {
     Object.assign(originalColors, newColors);
   };
 
+  (async () => {
+    // Custom colors for "Add an input/label" block icons in the "Make a block" popup menu, by pumpkinhasapatch
+    const uriHeader = "data:image/svg+xml;base64,";
+    const myBlocksCategory = categories[9];
+    while (true) {
+      // Wait until "Make a block" popup is opened and icon elements are created
+      const iconElement = await addon.tab.waitForElement("[class^=custom-procedures_option-icon_]", {
+        markAsSeen: true,
+      });
+      // Get img.src, remove data:image... header, then atob() decodes base64 to get the actual <svg> tags.
+      let svg = atob(iconElement.src.replace(uriHeader, ""));
+
+      // Find and replace the default color codes in the svg with our custom ones
+      svg = svg.replace("#ff6680", primaryColor(myBlocksCategory));
+      svg = svg.replace("#ff4d6a", secondaryColor(myBlocksCategory));
+      svg = svg.replace("#f35", tertiaryColor(myBlocksCategory));
+      svg = svg.replace("#fff", uncoloredTextColor()); // Text color for "Add a label" icon
```

Comment:
This might break with specific combinations of colors - see #7545.

Reply:
Oh I see, the color codes might replace each other if they contain similar values, like if one of the custom block colors starts with `#fff` then half of it would get replaced by the text color. It could use placeholder names like your PR so that doesn't happen.

The proper way to edit svg colors is by using class and style tags, but that's more work that doesn't need to be done unless something else changes these block icons for some reason.

Reply:
Using placeholders is probably a good enough solution.